src/poliastro/maneuver.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, *impulses):
 
         Notes
         -----
-        TODO: Fix docstring, *args convention
+        TODO: Fix docstring, \*args convention
```

Comment:
I meant \

Reply:
The backslash is needed because otherwise docutils thinks it's the start of an emphasis text that is never closed. It's a way to "escape" the star/asterisk.

Reply:
Got it! Everything good to go! Sorry I didn't know this